audit-ci.json

```diff
@@ -2,5 +2,5 @@
     "high": true,
     "pass-enoaudit": true,
     "retry-count": 20,
-    "allowlist": ["underscore"]
+    "allowlist": ["1067259", "node-forge", "moment"]
```

Comment:
Have these all been fixed in current master?

Reply:
No.    Current main lists 1067259 and node-forge.   The former is related to the saml library, and node-forge is due to downstream dependencies.   Our npm v8 update may allow us to force one or both of those via the 'new' override dependency setting, assuming it doesn't break our upstream code, but that should give us some time to migrate away from unmaintained packages or await an update. 

 https://github.com/nasa/cumulus/blob/master/audit-ci.json